cli/types/cypress.d.ts

```diff
@@ -2228,7 +2228,7 @@ declare namespace Cypress {
     })
     ```
      */
-    writeFile<C extends FileContents>(filePath: string, contents: C, options?: Partial<WriteFileOptions>): Chainable<C>
+    writeFile<C extends FileContents>(filePath: string, contents: C, options?: Partial<WriteFileOptions & Timeoutable>): Chainable<C>
```

Comment:
Note: readFile's option type already includes Timeoutable, but it is not used in the existing implementation beyond logging.

Reply:
It yields `null`, so is this...

```suggestion
    writeFile(filePath: string, contents: FileContents, options?: Partial<WriteFileOptions & Timeoutable>): Chainable<null>
```

?

Reply:
Oh good call, I'll get this and the other definitions updated.